### DISCUSSION TITLE
Support init / running storybook with deno 2

### DISCUSSION THREAD

**Discussion by birkskyum** [storybookjs/storybook#29360] [Ideas]
### Is your feature request related to a problem? Please describe.

I'd like to be able to init storybook and run it with deno. Recently released Deno 2 has node / npm compat, so it should be more feasible than in the past.

Ticket here, but I believe it's necessary to make changes in the storybook repo
- https://github.com/denoland/deno/issues/20610

### Describe the solution you'd like

`deno run -A npm:storybook@latest init`
`deno task storybook`

### Are you able to assist to bring the feature to reality?

no

**Comment by Qubut** [comment]
Any news about this one?
